fileshare/fileshare_startup/startup.go

```diff
@@ -23,9 +23,10 @@ import (
 	meshpb "github.com/NordSecurity/nordvpn-linux/meshnet/pb"
 )
 
+var ErrLAddressAlreadyInUse = errors.New("address already in use")
```

Comment:
I see it is being checked but nothing produces it. Is it used?

Reply:
It was produced by the `toError` helper func used for translating erorr codes coming from libdrop to our error representation. Now, libdrop provides actual errors so I deleted this `toError`. And this `ErrLAddressAlreadyInUse` was defined there ([here](https://github.com/NordSecurity/nordvpn-linux/pull/477/files#diff-72526275c385cbaa1d36587d90c8494c6ace084277d70c5fe235f24c0e3b2a8cL10))
It should be actually removed. I will use libdrop err [here](https://github.com/NordSecurity/nordvpn-linux/pull/477/files#diff-dcde5f61b97deab08ccf85fccbfeb0dd6308d47b30db676202080055d75bfe10L184)